self-core-impl/src/test/java/com/selfxdsd/core/contributors/ProjectContributorsTestCase.java

```diff
@@ -307,22 +308,23 @@ public void electsNewContributorForUnassignedTask() {
      * @return Contributor.
      */
     public Contributor mockContributor(
-        final String username, final String... roles
+            final String username, final String... roles
     ) {
         final Contributor contributor = Mockito.mock(Contributor.class);
         Mockito.when(contributor.username()).thenReturn(username);
 
         final List<Contract> contracts = new ArrayList<>();
-        for(final String role : roles) {
+        for (final String role : roles) {
             final Contract mock = Mockito.mock(Contract.class);
             Mockito.when(mock.role()).thenReturn(role);
             contracts.add(mock);
         }
 
         Mockito.when(contributor.contracts()).thenReturn(
-            new ContributorContracts(
-                contributor, contracts, Mockito.mock(Storage.class)
-            )
+                new ContributorContracts(
+                        contributor,
+                        contracts::stream,
+                        Mockito.mock(Storage.class))
         );
```

Comment:
@criske same here, indentation

Reply:
@amihaiemil I'll go through all the changed files and fix the bad indents, now that I updated the ide.